### DISCUSSION TITLE
Recent update (2 days ago) changed the way you call your instances?

### DISCUSSION THREAD

**Discussion by medledan** [TheLastBen/fast-stable-diffusion#1450] [Q&A]
The previous way was using a prompt like: 

((instance photo name))), showing face, full body, wearing t-shirt and pants, realistic, happy, dark red hair with highlights, 8k, highly detailed, natural light.

This always worked great and I have 1,000s of images I like. 

They changed the way you call your 'trainee" and now using the specific training name doesn't result in anything you trained (instance photo) wise showing up? Any ideas?

**Comment by TheLastBen** [comment]
I didn't change that, are you using the official colab or a fork ?

**Reply by medledan** [reply]
> I didn't change that, are you using the official colab or a fork ?

Oh duh, I'm sorry. This is in automatic1111. Wondering if you knew what was changed since a lot of the models trained via your process are used on Automatic.

**Reply by TheLastBen** [reply]
if you're talking about the extension, I'm sure you'll find the answer here https://github.com/d8ahazard/sd_dreambooth_extension
